templates/default.yaml

```diff
@@ -123,6 +123,19 @@ mountInotify: null
 # ADVANCED CONFIGURATION
 # ===================================================================== #
 
+# User to be used inside the VM
+user:
+  # User name. An explicitly specified username is not validated by Lima.
+  # 🟢 Builtin default: same as the host username, if it is a valid Linux username, otherwise "lima"
+  username: null
```

Comment:
`user.username` stutters

Reply:
In the code we have `user.Username` because `user.Name` is the GECOS, but the yaml can be simplified

Reply:
Can add a separate PR for the gecos, since it seems to be used in cloud.cfg (and simple enough in ci)

Reply:
* #2830

Reply:
Ok, will go with `name`, `home`, and `uid` for now, but what should we use for `gecos`?